cmd/substreams-sink-postgres/bulk_insert.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/streamingfast/cli"
+	. "github.com/streamingfast/cli"
+	"github.com/streamingfast/cli/sflags"
+	"github.com/streamingfast/shutter"
+	sink "github.com/streamingfast/substreams-sink"
+	"github.com/streamingfast/substreams-sink-postgres/db"
+	"github.com/streamingfast/substreams-sink-postgres/sinker"
+	"go.uber.org/zap"
+)
+
+var bulkLoadCmd = Command(bulkLoadE,
```

Comment:
The Go file for this command is `bulk_insert.go` but the command is using `load`. Let's normalize both. 

Also, I feel `bulk-load/insert` is not really appropriate as a name actually here since we have generating the `csv` file. Probably `generate-csv` or something along this, not quick sure the best name though.

Reply:
Updated to `generate-csv`, what do you suggest about the `BulkSinker`?

Reply:
`GenerateCSVSinker`